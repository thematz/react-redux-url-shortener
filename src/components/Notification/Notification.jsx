// src/components/Notification/index.jsx

import React from 'react'

const Notification = props => {
    const { type, message } = props
    return (
        <div className={"notification is-size-7 " + type}>
            {message}
        </div>
    )
}

export default Notification