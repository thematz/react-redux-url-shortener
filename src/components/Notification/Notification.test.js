import React from 'react'
import Notification from './Notification'
import store from '../../store'

import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
Enzyme.configure({ adapter: new Adapter() })

it('renders Notification correctly', () => {
    const wrapper = shallow(
        <Notification store={store} type='is-danger' message='Error' />,
    )
    expect(toJson(wrapper)).toMatchSnapshot()
})