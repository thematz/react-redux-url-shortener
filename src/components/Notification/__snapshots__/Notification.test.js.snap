// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Notification correctly 1`] = `
<div
  className="notification is-size-7 is-danger"
>
  Error
</div>
`;
